### PR TITLE
Fix visited gene coverage button link style

### LINKED ISSUE #3586: Gene coverage button ugly when clicked
image

### PR DESCRIPTION
**Pull request by northwestwitch**
fix #3586 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DNIL
- [x] tests executed by CR, DNIL

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/components.html [hunk 1]
```diff
@@ -81,11 +81,9 @@
           {% if config.SQLALCHEMY_DATABASE_URI %}
             <div class="d-flex flex-wrap">
               {% for gene in variant.genes %}
-                <span class="ms-5">
-                  <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
-                    Gene coverage:{{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
-                  </a>
-                </span>
+                <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
+                  Gene coverage: {{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/components.html:85]: Both colon and space! 😄

File: scout/server/blueprints/variant/templates/variant/components.html [hunk 2]
```diff
@@ -81,11 +81,9 @@
           {% if config.SQLALCHEMY_DATABASE_URI %}
             <div class="d-flex flex-wrap">
               {% for gene in variant.genes %}
-                <span class="ms-5">
-                  <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
-                    Gene coverage:{{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}
-                  </a>
-                </span>
+                <a class="btn btn-sm btn-secondary" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/components.html:84]:
```suggestion
                <a class="btn btn-sm btn-secondary text-white" target="_blank" href="{{ url_for('report.gene', gene_id=gene.hgnc_id, sample_id=variant.samples|map(attribute='sample_id')|list) }}">
```

Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/components.html:84]: Yes it doesn't work, but why the other links like igv and alamut do? Weird!

**Comment by northwestwitch** [comment]
Text-white did the trick:

image

Dark mode:

image

**Comment by dnil** [comment]
> Text-white did the trick:
> 
> image
> 
> Dark mode:
> 
> image

`role=button` or some css magic could do it as well, but hey, if it works.. 😁

**Review by dnil — APPROVED** [review]

**Pull request merged**